src/scenes/home/informationForm/informationForm.js

```diff
@@ -214,8 +214,7 @@ class SignupInformation extends Component {
         <div className={styles.buttonContainer}>
           <FormButton text="Go Back" onClick={this.previousPage} theme="blue" />
           <FormButton text="Save and Continue" onClick={this.saveAndContinue} theme="red" />
-          {this.state.error ? <ul className={styles.errorList}>There was an error saving your information, please try again.
-                              </ul> : null }
+          {this.state.error ? <ul className={styles.errorList}>There was an error saving your information, please try again.</ul> : null }
```

Comment:
This can be cleaned up to use the `&&` short circuit.
```javascript
{this.state.error && (
  <ul className={styles.errorList}>
    There was an error saving your information, please try again.
  </ul>
)}
```

Reply:
@mduleone  Can you please explain me a bit, I will make the changes.

Reply:
@reddysridhar53 In JS, because of how the boolean operators work, you can use `&&` or `||` to "short circuit" and cause certain lines of code to execute or not without an `if` statement by taking advantage of how javascript resolves.

See the below examples

```javascript
const log = (ish) => console.log(ish);
let testVar = false;

testVar && log('lolol'); // Nothing logged because `testVar` is false
testVar || log('lolol'); // 'lolol' logged because `testVar` is false
testVar = true;
testVar && log('lolol'); // 'lolol' logged
testVar || log('lolol'); // Nothing logged because `testVar` is true
```

We can take advantage of that when we're inside of JSX and manipulating things inside of `{}`. React doesn't render fasly values when it comes upon them when it's trying to render, so when it sees a falsy value from `this.state.error`, as it would after my suggestion (similar to what it does when it sees `null` with the ternary in the current code), it won't render anything.

Reply:
Ahh didn't see the updated comment, but I hope you liked the explanation!